### ISSUE TITLE
CI for PRs with mdl markdown linter

### ISSUE THREAD

**Issue by canweriotnow** [adlnet/xAPI-Spec#837]
Related: #829, #836.

Adding a separate issue to add Travis CI to run `mdl` on PRs pre-merge to ensure consistent and proper formatting... I'll set up another PR for the Travis config after the linter issues in #829 and #836 are resolved.

I think this is important as the contributor community grows, to keep the source clean and usable.

As @garemoko and I have discussed, this may involve creating a style doc (we should probably have a style guide in general, maybe on the wiki, but this is specific to testing) if there develops a consensus that some linter rules just aren't for us. That's maybe a convo that should happen here or on the mailing list.

**Comment by garemoko** [comment]
There's a style guide in [CONTRIBUTING.md](https://github.com/adlnet/xAPI-Spec/blob/1.0.3-Restructure/CONTRIBUTING.md#style-guide) that you can add to. 

In terms of order, it may make sense to set up Travis before doing the markdown fix PRs so that we can use Travis to check them. This does mean that every other PR will be failing until the markdown PRs land, but that's fine.

**Comment by canweriotnow** [comment]
@garemoko Agreed, I'll submit a separate PR w/ the Travis setup. I would be happy to take point on managing CI stuff, but I would need write access to the repo to set up the service integrations.

Funny you mention the style guide, I just put in #838 to track a minor error there :smile_cat:

**Comment by andyjohnson** [comment]
Same as issue #829 , this can wait until after the editor as needed.

**Comment by garemoko** [comment]
Propose closing this because I don't think it's going to get addressed.
